src/run_assistant_audio.cc

```diff
@@ -256,30 +255,32 @@ int main(int argc, char** argv) {
       if (response.has_audio_out()) {
         // CUSTOMIZE: play back audio_out here.
 
-        std::shared_ptr<std::vector<unsigned char>>
-            data(new std::vector<unsigned char>);
+        std::shared_ptr<std::vector<unsigned char>> data(
+            new std::vector<unsigned char>);
         data->resize(response.audio_out().audio_data().length());
         memcpy(&((*data)[0]), response.audio_out().audio_data().c_str(),
-            response.audio_out().audio_data().length());
+               response.audio_out().audio_data().length());
         audio_output.Send(data);
-
       }
       // CUSTOMIZE: render spoken request on screen
       for (int i = 0; i < response.speech_results_size(); i++) {
-        google::assistant::embedded::v1alpha2::SpeechRecognitionResult result =
-            response.speech_results(i);
+        auto result = response.speech_results(i);
```

Comment:
`auto` seems to have been introduced in C++11. Can we assume that as the earliest required version?

Reply:
According to the Makefile C++11 is the required version:
``
CXXFLAGS += -std=c++11 $(GRPC_GRPCPP_CFLAGS)
``